modules/text2vec-palm/clients/palm.go

```diff
@@ -206,9 +225,18 @@ func (v *palm) parseGenerativeAIApiResponse(statusCode int,
 
 	vectors := make([][]float32, len(resBody.Embeddings))
 	for i := range resBody.Embeddings {
-		vectors[i] = resBody.Embeddings[i].Value
+		if config.Model == "embedding-gecko-001" {
```

Comment:
please add a constant for the model or create a isLegacy() function

Reply:
also done :)